Makefile

```
@@ -280,15 +299,26 @@ bin/helm:
 	tar -zxvf $(TMP)/$(HELM_RELEASE) -C $(TMP)
 	mv $(TMP)/linux-amd64/helm bin/helm
 
-.PHONY: values.yml
-values.yml:
+bin/dep:
+	mkdir -p bin
+	curl -L https://github.com/golang/dep/releases/download/v0.5.0/dep-linux-amd64 > bin/dep
+	chmod +x bin/dep
+
+
+versions-values.yaml:
 ifndef RELEASE_STREAM
 	$(error RELEASE_STREAM is undefined - run using make values.yaml RELEASE_STREAM=vX.Y)
 endif
 	docker run --rm \
 	  -v $$PWD:/calico \
 	  -w /calico \
-	  ruby:2.5 ruby ./hack/gen_values_yml.rb $(RELEASE_STREAM) >> _includes/$(RELEASE_STREAM)/charts/calico/values.yaml
+	  ruby:2.5 ruby ./hack/gen_values_yml.rb $(RELEASE_STREAM) >> _includes/$(RELEASE_STREAM)/charts/calico/versions-values.yaml
```

```suggestion
	  ruby:2.5 ruby ./hack/gen_values_yml.rb $(RELEASE_STREAM) > _includes/$(RELEASE_STREAM)/charts/calico/versions-values.yaml
```

Instead of appending, let's just write it?